multicluster/controllers/multicluster/stale_controller.go

```diff
@@ -284,12 +362,21 @@ func (c *StaleResCleanupController) Run(stopCh <-chan struct{}) {
 	klog.InfoS("Starting StaleResCleanupController")
 	defer klog.InfoS("Shutting down StaleResCleanupController")
 
+	go wait.Until(c.runWorker, time.Second, stopCh)
 	if err := c.RunOnce(); err != nil {
 		c.Enqueue()
-		go wait.Until(c.runWorker, time.Second, stopCh)
 	}
 
-	<-stopCh
+	ticker := time.NewTicker(5 * time.Minute)
```

Comment:
This is not needed by member clusters? If so, could we just do periodic check for leader? @luolanzone 

As it will not happen in a normal case that MemberAccounce is left not deleted by the member cluster, I think it is even fine we run cleanup just once after leader cluster startup too.

Reply:
Yeah, If leader controller is running correctly, we can assume MemberClusterAccounce will be cleaned up during reconciling. I feel clean up once during startup is enough.

Reply:
Removed the ticker.